### PR TITLE
completed express-error-handling exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: express-error-handling/server/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  const { name, course } = req.body;
+  const score = Number(req.body.score);
+  if (!Number.isInteger(score) || score < 0 || score > 100) {
+    throw new ClientError(400, 'gradeId must be a positive integer');
+  }
+  if (!name || !course) {
+    res.status(400).json({
+      error: 'name, course, and score are required fields'
+    });
+    return;
```
Comment (scbowler) [inline express-error-handling/server/index.js:43]: Should be a `ClientError` here

**Review by scbowler — APPROVED** [review]
Looks great, just missed one `ClientError`. Let me know if you have any questions.

**Review by scbowler — CHANGES_REQUESTED** [review]
Ooops didn't mean to approve

File: express-error-handling/server/index.js (hunk 1 above)
Comment (avallonsmith) [reply express-error-handling/server/index.js]: Everything has been updated, apologies for the inconvenience!

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**